pages/k8s/install-aws.md

```diff
@@ -0,0 +1,266 @@
+    - image: busybox
+      command:
+        - sleep
+        - "3600"
+      imagePullPolicy: IfNotPresent
+      name: busybox
+      volumeMounts:
+        - mountPath: "/pv"
+          name: testvolume
+  restartPolicy: Always
+  volumes:
+    - name: testvolume
+      persistentVolumeClaim:
+        claimName: testclaim
+EOY
+```
+
+<div class="p-notification--caution">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+If you create EBS volumes and subsequently tear down the cluster, make sure to check
+with the AWS console to make sure all the associated resources have also been released.
+  </p>
+</div>
+
+### Using ELB Loadbalancers
+
+With the aws-integrator charm in place, actions which invoke a loadbalancer in
+Kubernetes  will automatically generate an AWS Elastic Load Balancer.  This can
+be demonstrated with a simple application. Here we will create a simple
+application running in five pods:
+
+```bash
+kubectl run hello-world --replicas=5 --labels="run=load-balancer-example" --image=gcr.io/google-samples/node-hello:1.0  --port=8080
+```
+
+You can verify that the application and replicas have been created with:
+
+```bash
+ kubectl get deployments hello-world
+ ```
+
+ Which should return output similar to:
+
+ ```bash
+ NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+ hello-world      5/5               5                            5             2m38s
+```
+
+To create a LoadBalancer, the application should now be exposed as a service:
+
+```bash
+ kubectl expose deployment hello-world --type=LoadBalancer --name=hello
+ ```
+
+To check that the service is running correctly:
+
+```bash
+kubectl describe service hello
+```
+
+...which should return output similar to:
+
+```yaml
+Name:                     hello
+Namespace:                default
+Labels:                   run=load-balancer-example
+Annotations:              <none>
+Selector:                 run=load-balancer-example
+Type:                     LoadBalancer
+IP:                       10.152.183.134
+LoadBalancer Ingress:     ad5fc7750350611e99768068a686bb67-239702253.eu-west-1.elb.amazonaws.com
+Port:                     <unset>  8080/TCP
+TargetPort:               8080/TCP
+NodePort:                 <unset>  31203/TCP
+Endpoints:                10.1.13.4:8080,10.1.13.5:8080,10.1.35.8:8080 + 2 more...
+Session Affinity:         None
+External Traffic Policy:  Cluster
+Events:                   <none>
+```
+
+You can see that the LoadBalancer Ingress is now associated with an ELB address in front
+of the five endpoints of the  example deployment. Leaving a while for DNS propagation, you
+can test the ingress address:
+
+```bash
+curl  http://ad5fc7750350611e99768068a686bb67-239702253.eu-west-1.elb.amazonaws.com:8080
+```
+```
+Hello Kubernetes!
+```
+
+Note that if you subsequently decommission this service, it is useful to check through
+the AWS console that the ELB resources are also released.
```

Comment:
I like the wording and call-out style for the EBS version of this message.  Would be nice to use the same here and just s/EBS/ELB/ or whatever.

Reply:
Yeah, i agree. I have changed it to look like the other one